Source/Particles/Collision/BinaryCollision/PhotonPhoton/TwoProductInitializeMomentum.H

```diff
@@ -0,0 +1,88 @@
+/* Copyright 2022 Remi Lehe
```

Comment:
Same comment as above:
```suggestion
/* Copyright 2023 Arianna Formenti
```